src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.php

```diff
@@ -21,6 +21,7 @@
 
 return static function (ContainerConfigurator $container) {
     $container->services()
+        // @deprecated since Symfony 6.3, to be removed in 7.0
```

Comment:
shouldn't this related only to the `addGlobalIgnoredName` call and not to the service definition ?

Reply:
This comment is meant to apply only to the next 2 lines:

```php
        ->set('annotations.reader', AnnotationReader::class)
        ->call('addGlobalIgnoredName', ['required'])
```

You're right, the service definition must be kept in the future. I'll change this, thanks.